src/ResourceManager/AzureBatch/documentation/upcoming-breaking-changes.md

```diff
@@ -25,20 +25,21 @@
     https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes/breaking-change-template.md
 -->
 
-# Upcoming Breaking Changes   
-
-## Release 5.0.0 - November 2017
-
-- `PSCloudPool` `ResizeError` was replaced by `ResizeErrors`.
-- `PSJobPreparationTaskExecutionInformation`, `PSJobReleaseTaskExecutionInformation`, `PSStartTaskInformation`, `PSTaskExecutionInformation`, and `PSSubtaskInformation` 
-  no longer have a `SchedulingError` property, they instead have a `FailureInformation` property. `FailureInformation` is returned any time there is a task failure.
-  This includes all previous scheduling error cases, as well as nonzero task exit codes, and file upload failures from the new output files feature.
+# Upcoming Breaking Changes
+
```

Comment:
The changes in the output objects are not covered here.  Please include a section on how the corresponding output object changes (*Nodes -> *ComputeNodes) would be handled by scripts that use the old proeprty values